bitrise.yml

```diff
@@ -53,12 +53,59 @@ workflows:
     before_run:
     - go-tests
     after_run:
+    - test_ndk
     - test_old_gradle
     - test_android-sdk21-tools-21.0.1
     - test_android-sdk20
     - test_android-sdk22-subdir
     - test_react_native
 
+  test_ndk:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-samples/android-ndk-sample-project.git
+    - BRANCH: master
+    - GRADLE_BUILD_FILE_PATH: ./build.gradle
+    - GRADLEW_PATH: ./gradlew
+    - ANDROID_NDK_HOME: ""
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
```

Comment:
do not mix `#!/usr/bin/env bash` and `#!/bin/bash`, prefer: `#!/usr/bin/env bash`